src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -78,11 +78,27 @@
      */
     private Map<String, DetectorData> detectors;
 
+    public Map<String, DetectorData> getDetectors(){
```

Comment:
also, move this function below your declarations

Reply:
Again, space after parentheses.